tests/phpunit/tests/filesystem/wpOpcacheInvalidateDirectory.php

```diff
@@ -37,7 +37,7 @@ public static function set_up_before_class() {
 	public function test_should_trigger_error_with_invalid_dir( $dir ) {
 		$this->expectError();
 		$this->expectErrorMessage(
-			'<code>wp_opcache_invalidate_directory()</code>',
+			'wp_opcache_invalidate_directory(): wp_opcache_invalidate_directory() expects a non-empty string.',
```

Comment:
`<code>...</code>` is expected here and shouldn't get stripped. But the tests are passing due to a bug in `wp_trigger_error()` HTML escaping. I have filed https://core.trac.wordpress.org/ticket/61318 to address the bug.

Once https://github.com/WordPress/wordpress-develop/pull/6671 is committed in trunk, this needs to be updated to 

```suggestion
			'wp_opcache_invalidate_directory(): <code>wp_opcache_invalidate_directory()</code> expects a non-empty string.',
```

Reply:
Thank you @thelovekesh, I will update this once the escaping issue is resolved 🗒️

Reply:
@prasadkarmalkar https://github.com/WordPress/wordpress-develop/pull/6671 is merged into the trunk. Please feel free to make your changes accordingly.

Reply:
Updated the test cases